### ISSUE TITLE
Please update tenant for ORCID (0000-0003-2106-4954)

### ISSUE THREAD

**Issue by LoriEP** [CDL-Dryad/dryad-product-roadmap#1054]
**Request:**
For user Paul Carvalho paulcarvalho@uri.edu & pcarvalh@ucsc.edu ORCID 0000-0003-2106-4954:
1. Please check to see if accounts need to be merged
2. Update tenant to University of Rhode Island

**Comment by sfisher** [comment]
This is a bit of a weird situation since I see three user accounts with his same first and last name.

It looks like he managed to successfully merge two accounts on his own with his former email address (our internal ids 66356 into 42467 with ORCID https://orcid.org/0000-0003-2106-4954 ).  The other account has his first and last name and no email (our id 66392) and is for this ORCID https://orcid.org/0000-0001-6354-2753 which shows it's been deleted or something weird on their web site.

Anyway, the two accounts that were already merged had two datasets in that account and I changed it to University of Rhode Island, so he should see that when he logs in.  I think it was UC Santa Cruz before.

The mystery ORCID account doesn't have any datasets in it, so I went ahead and deleted it.  It will just create it again if someone logs in with it again.

Thanks.

**Comment by LoriEP** [comment]
@sfisher this dataset payment record is showing as being sponsored by UCSC still, will that have to be fixed on the backend?

**Comment by LoriEP** [comment]
DOI: 10.7291/D1DM42

**Comment by LoriEP** [comment]
If I need a new ticket, please let me know, happy to create it. But affiliation seems to not be processing through, perhaps it is because the dataset was started with his old tenant record.

**Comment by sfisher** [comment]
Yes, this was submitted under UCSC which also gets free submissions like University of Rhode Island.

The accounts are merged and both datasets show up under his account. However, tenant associations at the dataset level usually stay with the institution that people are at when they submitted.  New dataset submissions for him will show up under the university of Rhode Island tenant in the database, but not updates for this one.  If he just wants to change his author affiliation metadata on a dataset there are no limitations on that in the metadata, I don't believe.

I don't think this effects what the admin people at University of Rhode Island see because they also see people associated by author affiliations (rather than dataset or former user affiliation).

We can move tenant associations in some cases for old datasets, but merging accounts doesn't do that, and not easily in this case because 1) The UCs are in a separate storage collection in Merritt and 2) UCSC uses EZID and most non-UCs use DataCite for their identifiers.  This means if we moved it, then updates wouldn't work properly for the data or for metadata because of different management (storage collection and DOI) for these two different institutions.

The first problem may be partially solved when the Merritt people complete a migration of the old UC data, but DOIs are supposed to be permanent and if we move it to be managed by DataCite (which is what URI uses) then they have different DOI prefixes and we'd need a new identifier managed by DataCite that would be different than the one there.

If this isn't published and he hasn't distributed his DOI far and wide, it might be easier to just re-submit it under URI and it will get the DataCite identifier instead of an EZID one.  But if it's published then we probably don't want to change the DOI and break everyone's links. IDK if the old dataset continuing to be managed under the UCSC is causing problems? Is it the author affiliation he's worried about--since that can be edited without the issues.

Have a good evening and I'll take a look tomorrow.

--Scott

**Comment by sfisher** [comment]
I had a look about the payment stuff and it still shows "payment sponsored by the UCSC" in the page because it is associated with that tenant still.

IDK if this is a big deal or not since they are both treated the same way for payment purposes.  It was changed in a different part of the database which may be used for other reporting, if that is the issue.

**Comment by LoriEP** [comment]
Thanks, I think it should be fine since I believe both are annual subscriptions/memberships.
